fbw-a380x/src/wasm/systems/a380_systems/src/structural_flex.rs

```diff
@@ -75,3 +87,1166 @@ impl SimulationElement for A380StructuralFlex {
         );
     }
 }
+
+struct A380WingLiftModifier {
+    lateral_offset: Ratio,
+
+    left_wing_lift: Force,
+    right_wing_lift: Force,
+
+    standard_lift_spread: Vector5<f64>,
+    lift_left_table_newton: Vector5<f64>,
+    lift_right_table_newton: Vector5<f64>,
+}
+impl A380WingLiftModifier {
+    const LATERAL_OFFSET_GAIN: f64 = 0.25;
+
+    // Ratio of the total lift on each wing section.
+    // Sum shall be 1.
+    const NOMINAL_WING_LIFT_SPREAD_RATIOS: [f64; 5] = [0., 0.42, 0.40, 0.15, 0.03];
+
+    // GAIN to determine how much a surface spoils lift when deployed. 0.3 means a fully deployed surface reduce lift by 30%
+    const SPOILER_SURFACES_SPOIL_GAIN: f64 = 0.4;
+    const AILERON_SURFACES_SPOIL_GAIN: f64 = 0.2;
+    const FLAPS_SURFACES_SPOIL_GAIN: f64 = 0.3;
+
+    fn default() -> Self {
+        assert!(Vector5::from(Self::NOMINAL_WING_LIFT_SPREAD_RATIOS).sum() == 1.);
+
+        Self {
+            lateral_offset: Ratio::default(),
+
+            left_wing_lift: Force::default(),
+            right_wing_lift: Force::default(),
+
+            standard_lift_spread: Vector5::from(Self::NOMINAL_WING_LIFT_SPREAD_RATIOS),
+            lift_left_table_newton: Vector5::default(),
+            lift_right_table_newton: Vector5::default(),
+        }
+    }
```

Comment:
Move to `Default` trait implementation.

Reply:
Too much time without any line of Rust :D